src/ui/components/NetworkMonitor/utils.ts

```diff
@@ -121,6 +138,7 @@ export const partialRequestsToCompleteSummaries = (
       const documentType = getDocumentType(response.event.responseHeaders);
       const type: RequestType = (documentType?.split("/")?.[1] || documentType) as RequestType;
       return {
+        cause: request.event.requestCause as RequestType,
```

Comment:
Since we're casting this, this could introduce unexpected keys. Maybe it would be better to have this be
```
cause: REQUEST_TYPES.hasOwnProperty(request.event.requestCause) ? request.event.requestCause : "other",
```
then we don't need the null check below at all.

Reply:
Yeah, I think I can write this line better now!

Reply:
Actually, yeah, because this type comes from protocol let's do it the ternary way 👍